app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,12 @@
 class PostsController < ApplicationController
   def index
     @user = User.find(params[:user_id])
-    @posts = @user.posts
+    @posts = @user.posts.includes([:author])
```

Comment:
- [ ] please like the previous reviewer noted, kindly  make sure you also use CanCanCan for Post authorization in Post Controller as required for the project.
eg 

 ```
 class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end

```

Reply:
done